src/Pages/kiwanlee/Main/Main.js

```diff
@@ -0,0 +1,188 @@
+import React from 'react';
+
+import './Main.scss';
+import Nav from '../../../Components/Nav/Nav';
+//클래스형 표현
+
+class Main extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      noDisplaying: 'none',
+      displaying: 'inline-block',
+      comment: '',
+      commentsList: [],
+    };
+  }
+  //하트채우기
+  heartLiked = () => {
+    if (this.state.displaying === 'inline-block') {
+      this.setState({ displaying: 'none' });
+      this.setState({ noDisplaying: 'inline-block' });
+    } else {
+      this.setState({ displaying: 'inline-block' });
+      this.setState({ noDisplaying: 'none' });
+    }
+  };
```

Comment:
```suggestion
  heartLiked = () => {
    this.setState({
      display: !this.state.display,
    });
 };
```
하트 채우기 함수도 클릭시 채워지고 비워지고를 반복하므로 true, false의 형태가 반복되게끔 로직을 짜는 것이 더 효율적입니다!